test/unit/model/cygwin-test.js

```diff
@@ -245,7 +245,6 @@ describe('Cygwin installer', function() {
         installer.detectExistingInstall(function() {
```

Comment:
This is the call that needs a return statement to be executed properly. I guess there are other in the code as well. Can you take a look?